src/new/wallet.js

```diff
@@ -2322,6 +2323,53 @@ class HathorWallet extends EventEmitter {
     const tx = await this.prepareCreateNewToken(name, symbol, amount, newOptions);
     return this.handleSendPreparedTransaction(tx);
   }
+
+  /**
+   * Sign all inputs
+   *
+   * @param {Transaction} tx The transaction to be signed
+   * @param [options]
+   * @param {string} [options.pincode] PIN to decrypt the private key.
+   *                                   Optional but required if not set in this
+   *
+   * @returns {Transaction} The signed transaction
+   */
+  getSignatures(tx, { pinCode = null } = {}) {
```

Comment:
Maybe we should call this method `signTx` because we are not really returning the signatures, we are actually signing the transaction and returning it already signed.

What do you think?

Reply:
Changed to `signTx` but created another `getSignatures` that actually return the signatures.